provisioning/device/src/constants.ts

```diff
@@ -23,7 +23,7 @@ export class ProvisioningDeviceConstants {
   /**
    * apiVersion to use while communicating with service.
    */
-  static apiVersion: string = '2018-11-01';
+  static apiVersion: string = '2019-01-15';
```

Comment:
> static apiVersion: string = '2019-01-15'; [](start = 1, length = 42)

has this API been deployed to all regions?  if not, this  change could break users.